katas/content/superdense_coding/index.md

```diff
@@ -0,0 +1,30 @@
+# Superdense Coding Kata
+
+@[section]({
+    "id": "./superdense_coding.ipynb__overview",
+    "title": "Overview"
+})
+
+**Superdense Coding** quantum kata is a series of exercises designed to get you familiar with programming in Q#.
+
+It covers the superdense coding protocol which allows us to transmit two bits of classical information by sending just one qubit using previously shared quantum entanglement.
+
+- A good description can be found in [the Wikipedia article](https://en.wikipedia.org/wikiSuperdense_coding).
+- A great interactive demonstration can be found [on the Wolfram Demonstrations Project](http:/demonstrations.wolfram.com/SuperdenseCoding/).
+- Superdense coding protocol is described in Nielsen & Chuang, section 2.3 (pp. 97-98).
```

Comment:
I would drop this one - it's not a link, and we cannot assume the reader bought a book too.
```suggestion
```

Reply:
Done, in earlier Katas, we were referring this book a lot. I like this book, it really explains some of the concepts in great details.